src/_doesCrossjoinYieldAllCombinationsOfMembers.ts

```diff
@@ -0,0 +1,29 @@
+import { Cube, LevelCoordinates } from "@activeviam/data-model";
+import {
+  findLevels,
+  isMdxCompoundIdentifier,
+  MdxFunction,
+} from "@activeviam/mdx";
+
+/**
+ * Returns whether `crossjoin` yields all combinations of members of the hierarchies expressed on the axis.
+ */
+export function _doesCrossjoinYieldAllCombinationsOfMembers(
+  crossjoin: MdxFunction,
+  { cube, levelsOnAxis }: { cube: Cube; levelsOnAxis: LevelCoordinates[] }
+): boolean {
+  const levelsInCrossjoin = findLevels(crossjoin, cube);
+  // Each level on the axis is represented in the crossjoin, and not only by a specific member of this level.
+  // For example:
+  // [Geography].[City].[AllMember].[Paris] // does not count
+  // [Geography].[City].Members // does count
+  return levelsOnAxis.every(({ dimensionName, hierarchyName, levelName }) =>
+    levelsInCrossjoin.some(
+      (levelInCrossjoin) =>
+        levelInCrossjoin.dimensionName === dimensionName &&
+        levelInCrossjoin.hierarchyName === hierarchyName &&
+        levelInCrossjoin.levelName === levelName &&
+        !isMdxCompoundIdentifier(levelInCrossjoin.match)
```

Comment:
This syntax seems to work actually, in order to retrieve all combination of the members of different levels. See screenshot below. Taking care of it is not complicated. I will make the change and add a test.

![image](https://user-images.githubusercontent.com/5601730/147237575-52798f47-b6d6-4f6f-a8df-f51ec162e51c.png)

Reply:
> This syntax seems to work actually
> I don't believe that AUI4 generates this kind of MDX

Yes that's what I meant^^ It's valid so it's nice to support it, although I wouldn't expect to see it much in real life, and it would be custom so there is an "excuse" if it does not work properly when it it migrated.